src/css/components/_progress-bar.scss

```diff
@@ -1,6 +1,7 @@
 .progress-bar {
     position: relative;
     height: calc((#{$master-height} - 75px) / 6) ;
+    color: inherit;
```

Comment:
Looks like this doesn't work for the light theme, though if we only include this property in the dark theme everything works fine
```
.dark-theme .progress-bar {
    color: inherit;
}
```